packages/management/framework/package.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: framework.tanzu.vmware.com
+  namespace: framework
```

Comment:
This seems to be a new namespace. Is there a consensus on using this new namespace? Also, who creates this namespace?

Reply:
this is just a placeholder value that got scaffolded when I created the `framework` package.. removed this just to not create any confusion.. need to do some cleaning in the scaffolding templates, will create a new pr for that